src/Connections/GitlabRepo.ts

```diff
@@ -512,6 +566,26 @@ ${data.description}`;
         }
         return false;
     }
+
+    public async provisionerUpdateConfig(userId: string, config: Record<string, unknown>) {
+        const validatedConfig = validateState(config);
+        await this.as.botClient.sendStateEvent(this.roomId, GitLabRepoConnection.CanonicalEventType, this.stateKey, validatedConfig
+        );
```

Comment:
Was the linter's max-len limit reached for the last two characters?
```suggestion
        await this.as.botClient.sendStateEvent(this.roomId, GitLabRepoConnection.CanonicalEventType, this.stateKey, validatedConfig);
```